Description: Add new IDs to device DB.
 Google Xoom 18d1:70a8 (LP: #793064)
Origin: https://launchpad.net/bugs/793064
Forwarded:
 https://sf.net/tracker/?func=detail&aid=3327059&group_id=158745&atid=809064
---
 src/music-players.h |    2 ++
 1 file changed, 2 insertions(+)

--- libmtp.orig/src/music-players.h
+++ libmtp/src/music-players.h
@@ -1247,6 +1247,8 @@
       DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST |
       DEVICE_FLAG_BROKEN_SET_OBJECT_PROPLIST |
       DEVICE_FLAG_BROKEN_SEND_OBJECT_PROPLIST },
+  { "Google", 0x18d1, "Xoom (ID 3)", 0x70a8,
+     DEVICE_FLAG_BROKEN_MTPGETOBJPROPLIST },
 
   /*
    * Media Keg
